go/exocom/features/configuring-the-ports.feature

```diff
@@ -0,0 +1,24 @@
+Feature: configuring the port
+
+  As an Exosphere developer testing my application
+  I want to be able to boot ExoCom at various ports
+  So that I have flexibility in testing my system.
+
+  - the default ExoCom port is 3100
+  - provide the "EXOCOM_PORT" environment variable to listen for HTTP requests on a custom port
+
+
+  Scenario: running at the default port
+    When I run ExoCom
```

Comment:
The terminology I found useful is:
- `When starting XXX` means the process is started, the step doesn't wait for anything, and the process is still running when the step finishes
- `When running XXX` means the process is started and the step waits until the process finished without an error code

There is also `When trying to run XXX`, which starts the process, waits until it finishes, and verifies that it exits with an error code.